substrate/primitives/api/proc-macro/src/utils.rs

```diff
@@ -334,6 +334,88 @@ pub fn get_deprecation(crate_: &TokenStream, attrs: &[syn::Attribute]) -> Result
 		.unwrap_or_else(|| Ok(quote! {#crate_::metadata_ir::DeprecationStatusIR::NotDeprecated}))
 }
 
+/// Represents an API version.
```

Comment:
I didn't change any of this code; just moved it to the utils module so that I could use it in runtime_metadata too.